_includes/header.html

```diff
@@ -30,11 +30,12 @@
     <div class="menu-button"><i class="icon icon-reorder"></i><span>Menu</span></div>
     <div class="nav-menu-on" role="navigation">
       <ul class="small-nav">
-        {%- include nav_item.html text="News" href="/blog" url_full="/blog/" url_fragment="blog" -%}
-        {%- include nav_item.html text="Source" href="/source.html" url_full="/source.html" url_fragment="source" -%}
+        {%- include nav_item.html text="Download" href="/downloads.html#opensearch" url_full="/downloads.html#opensearch" url_fragment="downloads" -%}
```

Comment:
Because this directory structure is part of the [Project Website](https://github.com/opensearch-project/project-website) repo and not the documentation website, we might just have to use the `https` schema as a work around for the time being, similar to how we link to Forums.

- Downloads: https://opensearch.org/downloads.html#opensearch 
- Blog: https://opensearch.org/blog/
- Events: https://opensearch.org/events/
- Partners: https://opensearch.org/partners/

Reply:
Since the forum in on a subdomain, we had to fully qualify the URL for the website header as well - this should be good to go